aws/aws-how-to/instances/launch-ubuntu-desktop.rst

```diff
@@ -1,78 +1,105 @@
+Use the xrdp server and Remmina to connect to an AWS EC2 instance of Ubuntu Mantis.
 
-.. Note::
+Select and configure Ubuntu Mantic 
+----------------------------------
 
-    Both the remote EC2 instance and your local machine are assumed to be running Ubuntu 16.04 or greater.
+Open AWS Marketplace and search for Ubuntu Mantic. Subscribe and agree to the Terms and Conditions. Select Launch Instance to configure.
 
+Select a region closest to you. For exapmple, if you're in the ``UK EU-West-2`` would be a good option. Click :guilabel:`Continue to Launch Through EC2 Instance`.
 
-Install Ubuntu desktop and TightVNC on your VM
-----------------------------------------------
+Give your instance a name. Mantic requires least two cores and at least 8gb RAM, which should be the default option. The default volume space of 8gb is the minimum required, however, more space will be required to install applications.
 
-To install Ubuntu desktop and TightVNC server on your EC2 instance, SSH into it and run:
+Create a security group
+-----------------------
+
+Select :guilabel:`Create Security Group`. Make sure the ssh tick box is selected to allow remote access.
+
+You can select an existing key pair if you already have one set up or create a new key pair. To create a key pair give it a name and Select ``RSA`` and ``PEM``. A Pem file will be automatically downloaded.
+
+Launch the instance and connect to it either via ssh or the AWS Console.
+
+Install Ubuntu Desktop Packages
+-------------------------------
+
+.. code::
+
+    sudo apt update && apt upgrade -y
+    sudo apt install -y ubuntu-desktop
+    sudo snap install snap-store --edge
+
+Install and Configure RDP
+-------------------------
+
+Install the xrdp server.
 
 .. code::
 
-    sudo apt update
-    sudo apt install ubuntu-desktop
-    sudo apt install tightvncserver
-    sudo apt install gnome-panel gnome-settings-daemon metacity nautilus gnome-terminal
+    sudo apt install -y xrdp
 
-Save the password created during the installation of the VNC server.
+Configure xrdp to use SSL to get an encrypted connection.
 
+.. code::
 
-Configure the VNC server
-------------------------
+    sudo usermod -a -G ssl-cert xrdp
 
-On your VM, launch the VNC server to create an initial configuration file:
+Set up a password for the Ubuntu user.
 
 .. code::
 
-    vncserver :1
+    passwd
 
-Edit the configuration file ``~/.vnc/xstartup`` to include:
+Finally, restart the xrdp service.
 
 .. code::
 
-    #!/bin/sh
+    systemctl restart xrdp
+
+Configure the Ubuntu Session
+----------------------------
 
-    export XKL_XMODMAP_DISABLE=1
-    unset SESSION_MANAGER
-    unset DBUS_SESSION_BUS_ADDRESS
+Connect to your instance using RDP to check the previous steps were succseful. Create a configuration script called ``ubuntu-session`` in ``/usr/local/bin/`` to run on RDP connections.
```

Comment:
typo: succseful (Run ``PROJECT=aws make spelling`` on your local instance to catch the typos)
It might be a good idea to mention that on successful connection, you only see a vanilla Gnome desktop without the Ubuntu session.